app/src/sharedTest/java/org/oppia/android/app/home/HomeActivityTest.kt

```diff
@@ -783,9 +808,9 @@ class HomeActivityTest {
   fun testHomeActivity_firstTestTopic_topicSummary_topicNameIsCorrect() {
     launch<HomeActivity>(createHomeActivityIntent(internalProfileId1)).use {
       testCoroutineDispatchers.runCurrent()
-      scrollToPosition(position = 2)
+      scrollToPosition(position = 3)
```

Comment:
Here and elsewhere: why do these positions need to be updated?

Reply:
Initially when there are no topic's in progress, recommended stories are shown by default. So the all topics list item position is moved down by 1position.

Reply:
Ditto else where.